### PR TITLE
Add curative fetcher

### PR DESCRIPTION
**Pull request by rajbot**
Run with `poetry run vaccine-feed-ingest fetch us/curativeinc_com`

Writes one json file per location.

### CODE REVIEW

File: vaccine_feed_ingest/runners/us/curativeinc_com/fetch.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    if num_not_found != run:
+        return False
+
+    return True
+
+
+site_id = 1
+while True:
+    url = f"{base_url}/{site_id}"
+    logger.info(f"Fetching {url}")
```
Comment (eliblock) [inline vaccine_feed_ingest/runners/us/curativeinc_com/fetch.py:55]: Let's remove this line - it makes for very noisy logs given the number of sites polled.
```suggestion

```

**Review by eliblock — APPROVED** [review]
Thank you for writing this @rajbot!

I've noticed a few other fetchers use `asyncio` to speed up fetches that involve a lot of network calls. Is it worth adding that here?

**Comment by obra** [comment]
What's this blocking on?

**Review by bryanculbertson — CHANGES_REQUESTED** [review]
This PR is pretty old at this point. It will need to be rebased to main and have the tests re-run.

It also sounds like it is really slow according to @eliblock 's comment?

**Comment by MoralCode** [comment]
it looks like the api url here is for testing sites (`https://labtools.curativeinc.com/api/v1/testing_sites`). Is this something that is still needed to add to the map? It is my impression that the map (and this dataset) is just focused on vaccination sites.

**Comment by obra** [comment]
That data is pretty clearly vaccination clinics. Also, curative's
data includes whether a site is a vaccination clinic or a testing site in
the per-site info.

**Comment by eliblock** [comment]
Closing all prs: https://blog.vaccinateca.com/vaccinate-the-states-is-winding-down

**Pull request closed without merging**